app/views/module_sections/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Module:</strong>
+  <%= @module_section.module_id %>
+</p>
+
+<p>
+  <strong>Module name:</strong>
+  <%= @module_section.module_name %>
+</p>
+
+<table>
+  <thead>
+    <tr>
+      <th>Page</th>
+      <th>Module</th>
+      <th>Content</th>
+      <th>Page name</th>
+      <th>Page description</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @pages.each do |page| %>
+      <tr>
+
+        <% if page.module_id == @module_section.module_id %>
```

Comment:
Clever use of the 'if' statement, do we need to include the 'else' for it to work?

Reply:
Based on our definition of done for modules, it fits. The pages each have a module and can be arranged so that each page has certain modules with it

Reply:
Way to follow our coding standards!